js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -280,7 +281,7 @@ export const AssetsOverview = ({viewerName}: {viewerName?: string}) => {
   }
 
   return (
-    <>
+    <Box flex={{direction: 'column'}} style={{height: '100%', overflow: 'hidden'}}>
```

Comment:
why are we using overflow: 'hidden'?
I think we want flex-grow here.

Reply:
ah true, this isn't necessary

(copied it over from the asset catalog page)